serializable/__init__.py

```diff
@@ -40,15 +39,16 @@
 else:
     from typing_extensions import Protocol  # type: ignore
 
+from ._logging import _LOGGER, _warning_kwargs
 from .formatters import BaseNameFormatter, CurrentFormatter
 from .helpers import BaseHelper
 
 # !! version is managed by semantic_release
 # do not use typing here, or else `semantic_release` might have issues finding the variable
 __version__ = '0.12.0'
 
-logger = logging.getLogger('serializable')
```

Comment:
bring back `logger` and mark it as depricated.
 even though `logger` was not used for any logging in the past, it still was public API.

Reply:
or leave as is and consider it a breaking change -- which is okay for a planned v1.0.0